server/main.lua

```diff
@@ -61,3 +62,73 @@ RegisterNetEvent('ox_police:setPlayerEscort', function(target, state)
 
     target:set('isEscorted', state and source, true)
 end)
+
+lib.callback.register('ox_police:spawnVehicle', function(source, model, mock)
+
+    -- perform checks to prevent exploits and abuse
+    local player = players[source]
+    local isClose = false
+    local isPolice = player.hasGroup(Config.PoliceGroups)
+
+    if not isPolice then
+        return print('player is not a policeman')
+    end
+
+    for i=1, #Config.SpawnerCoords do
+        local spawnerCoords = Config.SpawnerCoords[i].coords
+        local distance = #(spawnerCoords - player.getCoords(true))
+        if distance <= 5.0 then
+            isClose = true
+            break
+        end
+    end
+
+    if not isClose then
+        return print('player is too far away')
+    end
+
+    if not player then
+        return print('player is not in service')
+    end
+
+    if type(mock) ~= "table" then
+        return print('invalid mock, possible cheater')
+    end
+
+    if not mock then
+        return print('missing mock, possible cheater')
+    end
```

Comment:
Line 98 won't run since type(mock) would catch nil or boolean; but I also don't see the need for it to exist.